src/main/java/cloud/klasse/backendbusiness/classroom/Classroom.java

```diff
@@ -0,0 +1,28 @@
+package cloud.klasse.backendbusiness.classroom;
+
+import cloud.klasse.backendbusiness.teacher.Teacher;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+
+@Entity
+@Table(name = "Classroom")
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Classroom {
+
+    private Long id;
+
+    private String topic;
+
+    private Teacher teacherId;
```

Comment:
Shouldn't this attribute teacher be called teacher?

Reply:
Fixed.